### DISCUSSION TITLE
Unable to reach Scrypted web UI via port 10443

### DISCUSSION THREAD

**Discussion by Bmkenny** [koush/scrypted#767] [Q&A]
Hi, I am not sure what happened, but I am suddenly not able to access scrypted web UI via my IP and port 10443. I ran the scrypted in the documentation for the recommended docker method and previously had portainer.io running as well. This is running on a raspberry pi 4b in conjunction with Hoobs. 

I get an error that my IP refused to connect and err_connection_refused. Also when I run the command, systemctl status scrypted.service, in terminal I get this, Unit scrypted.service cannot be found. 

Can you please help so I can get scrypted back up and running?! Thank you!

**Comment by Bmkenny** [comment]
@koush any insights by chance? 

Forgot to mention in my original post that my scrypted instance seems to still be active as I can see my ring cameras in HomeKit and can set HSV settings up but there is no video feed. 

Any help would be greatly appreciated! Thanks!

**Reply by Bmkenny** [reply]
@koush or anyone can anyone help? I'd really like to get my Scrypted instance back up and running fully and have the video feeds working in my apple home app/homekit ASAP. Thanks.

**Reply by Franchesco49** [reply]
I have de same problem here and looks like there is some authorization issues. We probably need to wait for a solution from Koush to understand what happened.

**Comment by Tarnor** [comment]
Just woke up this morning and I have a similar problem.  If I run a docker ps -f name=scrypted, I see that it's only running for a small number of seconds each time so I'm guessing it's restarting itself for some reason before getting to the web interface.

Will post again if I can fix it.

**Reply by koush** [reply]
if you're on a pi, try running `sudo apt dist-upgrade`

**Reply by pschroeder89** [reply]
This worked for me. I also needed to pull the latest docker image and restart my portainer  stack

**Reply by Tarnor** [reply]
Thanks.  I've tried dist-upgrade with no luck. Any chance you can share the commands you used to pull the latest image and restart the stack?  I'm a docker noob.   Thanks so much!

**Reply by NebuNC2** [reply]
I have the same issue since last night.
dist-upgrade didn’t work for me aswell.

**Reply by Franchesco49** [reply]
Same problem here and try the solution but didn't solve the issue. I'm using a Pi4 with docker composed way.

**Reply by NebuNC2** [reply]
Now it works again. Without doing anything

**Reply by pschroeder89** [reply]
Run `docker pull koush/scrypted`. Then just restart your machine and it should use the new image when it reboots.

**Reply by Tarnor** [reply]
Pulled the new image... seemed to install fine, but still nothing after rebooting.
Looking at the docker container it seems like scrypted is only alive for a minute or so and then it restarts.

**Reply by koush** [reply]
please make sure you're on docker 23. I'm guessing you're on an old ubuntu distribution and/or old docker.

**Reply by Tarnor** [reply]
Ok interesting.

Looks like docker 24.0.2, build from May 25th.
And Raspberry Pi Bullseye - Raspbian GNU/Linux 11

Currently I seem to have 2 scrypted images installed though:
TAG: latest from 28 hours ago
TAG: none with image ID 264077355e8b from 32 hours ago.  (And 32 hours ago was exactly when my cameras went offline).
I also have the watchtower image installed (latest) from 4 months ago.

Running Docker Stats, I get that scrypted is running, but consuming 0bytes, 0CPU, 0 memory etc.  
Watchtower is also running, and consuming a small amount of memory.

**Reply by Franchesco49** [reply]
> Run `docker pull koush/scrypted`. Then just restart your machine and it should use the new image when it reboots.

Thanks @pschroeder89 , it works fine now ! My wife with so upset not to see her baby girl through HK !

**Reply by Franchesco49** [reply]
I have another issue. All my cameras (with Hikvision and RTSP. I can see live view on HK but I don't have any record on HKSV since Sunday. Any idea where the problem can come ? Thanks for your help !

**Reply by jshenabc** [reply]
I got the same situation, do you happen to know the solution now?

**Comment by one800J** [comment]
Remove this from the yml entrypoint.  

 

/init

**Reply by Tarnor** [reply]
I see running the docker ps -f name=scrypted command that the IMAGE COMMAND is "/init /bin/sh.....", but how can I remove the "/init? part to give it a go?

The STATUS is showing "Restarting (159) 23 seconds ago ", and seems to be restarting constantly since Saturday's update.

Thanks for your help.

**Reply by jshenabc** [reply]
@Tarnor do you have the solution now?

**Reply by Tarnor** [reply]
I uninstalled it, and reinstalled it again.  Had to set up the cameras from scratch, but that was the only thing that got it working for me.  Sorry I don't have a better solution.
